typings/pinejs-client-core.d.ts

```diff
@@ -1,3 +1,29 @@
+import { PropsOfType } from './utils';
+
+export interface WithId {
+	id: number;
+}
+
+export interface Deferred {
+	__id: number;
+}
+
+/**
+ * When not selected-out holds a deferred.
```

Comment:
NIT What does 'not selected-out' mean? Can you be more verbose here?

Reply:
Done. Removed the comment from here and updated the one in the re-exported interface found in the SDK, so that we get those helpful description tooltips on hover.